embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/Embrace.java

```diff
@@ -515,6 +516,24 @@ public boolean recordCompletedSpan(@NonNull String name, long startTimeNanos, lo
         return false;
     }
 
+    /**
+     * Adds one or many [SpanExporter] to the tracer.
+     *
+     * @param spanExporter the span exporters to add
+     */
+    public void addSpanExporter(SpanExporter... spanExporter) {
```

Comment:
Lets define this in this in the `EmbraceApi`? Everything public should be in one of the interfaces.

And what do you think about just taking one exporter in the parameter? It's unlikely that ppl will want more than one, and if they do, they can create their own composite or just call this method again. This reduces the complexity and surface area of the API which would be preferable.